MobiFlight/Board.cs

```diff
@@ -279,6 +328,37 @@ public class Board : IMigrateable
         /// </summary>
         public UsbDriveSettings UsbDriveSettings;
 
+        /// <summary>
+        /// Base path for custom firmware
+        /// </summary>
+        public String BasePath;
+
+        /// <summary>
+        /// The image resource
+        /// </summary>
+        public Image BoardImage;
+
+        /// <summary>
+        /// Returns the correct Partner Level
+        /// </summary>
+        /// <returns></returns>
+        [JsonIgnore]
+        public BoardPartnerLevel PartnerLevel { get {
+                if (BasePath == "" && Info.MobiFlightType.Contains("MobiFlight"))
+                    return BoardPartnerLevel.Core;
+
+                var partners = new List<string>() { 
+                    "kavSimulations",
+                    "miniCOCKPIT"
+                };
```

Comment:
I think this should move out to a top-level property on the class at the top of the file to make it easier to find in the future when needing to update the list. Right now it's hiding inside a method.